### ISSUE TITLE
Underlying requests access mechanism is not working on every request

### ISSUE THREAD

**Issue by MLDVN** [ultrafunkamsterdam/undetected-chromedriver#177]
I truly enjoy using this library, as it is way better configured than the standard selenium or even than selenium-wire, but it lacks a layer for accessing the requests since the standard mechanism provided with selenium is in some cases intercepted by the websites and some of the requests are not returned via desired capabilities way of logging. Is it possible to implement a way of handling the requests similar to what selenium-wire provides? I would be forever grateful for something like this, looking forward for your reply, thanks alot.

**Comment by ultrafunkamsterdam** [comment]
So what you want is an intercepting proxy showing the actual bytes flowing through the stream? That shouldnt be too hard. Ill see what i can do.

**Comment by MLDVN** [comment]
Yes, I guess working on the level of bytes would work in this case, my particular use case that I have in mind requires me to check the response that a requests gets when I make it from the browser and I can only do it from selenium, because the target of the requests has some kind of detection layer that will block regular requests and will also hide it from selenium. I hope it makes sense, will follow this thread. Thanks again

**Comment by HereForTraffic** [comment]
Any update on this issue? Would love to see it solved, tyvm!

**Comment by montovaneli** [comment]
The selenium-wire docs (https://pypi.org/project/selenium-wire/#bot-detection) says that it automatically integrates with undetected-chromedriver if it finds it in your environment. So I really didn't understand your question.

**Comment by MLDVN** [comment]
From what I observed in my tests and uses cases, selenium-wire is not as well configured and it gets easily detected even when used with undetected-chromedriver, that`s why I asked for the feature here to circumvent adding a supplemental dependency.

**Comment by alemosa** [comment]
any news on this? I also need a way to intercept requests and add custom headers to them, but the selenium-wire library with uc gets easily detected...
